api/src/main/scala/uk/ac/wellcome/platform/api/controllers/WorksController.scala

```diff
@@ -130,4 +132,14 @@ class WorksController @Inject()(@Flag("api.prefix") apiPrefix: String,
     }
 
   }
+
+  /// Parse a comma-separated ?includes query parameter.
+  ///
+  /// TODO: Throw an exception if we get an unrecognised include.
```

Comment:
I’m not planning to address this TODO in this patch – keep the refactor and feature addition separate!